Apps/Sandcastle/gallery/Particle System.html

```diff
@@ -278,13 +278,15 @@
     rate : viewModel.rate,
 
     bursts : [
-        new Cesium.ParticleBurst({time : 5.0, minimum : 300, maximum : 500}),
+        // these burst will occasionally sync to create a multicolored effect
+        new Cesium.ParticleBurst({time : 5.0, minimum : 10, maximum : 100}),
         new Cesium.ParticleBurst({time : 10.0, minimum : 50, maximum : 100}),
         new Cesium.ParticleBurst({time : 15.0, minimum : 200, maximum : 300})
     ],
 
     lifeTime : 16.0,
 
+    //emitter : new Cesium.SphereEmitter(5.0),
     emitter : new Cesium.CircleEmitter(0.5),
```

Comment:
If we're starting with the `Sphere` emitter selected, shouldn't this be a `Sphere` and not `Circle`?